### ISSUE TITLE
MSI Mainboard LEDs

### ISSUE THREAD

**Issue by dilution1** [Vipeax/MSI-LED-Tool#7]
Hi!

I'd like  to know if you can also change the colours of the LEDs on the mainboard with yout tool (my mainboard is a MSI Z170A Gaming Pro Carbon). If not, is it possible to add this feature to your tool or create a new tool, which can do that? Because I really don't want to use the Gaming App.

Thank you

**Comment by Vipeax** [comment]
This tool only focuses on the GPU. I suppose it would be possible to make one for motherboards as well, but I don't own any MSI hardware myself so when I created the GPU variant others tested it for me. In this case you can be a test dummy if you want... 🚀

**Comment by dilution1** [comment]
Hi!
What could happen at worst if i am your test dummy? Can my mainboard get damaged? If nothing can happen i will be your test dummy :)

**Comment by Vipeax** [comment]
Not really. It's a bit difficult to explain, but breaking your motherboard is pretty much impossible. I just need someone to confirm that it does what it should do.

**Comment by dilution1** [comment]
Ok. Then i will be your man. Just tell me what to do :)

**Comment by Vipeax** [comment]
I'm currently out of the country. I might have something for you to test somewhere during the weekend.

**Comment by dilution1** [comment]
Alright. Thank you!

**Comment by brakim** [comment]
This feature would be great, i can also help to test a new program.
